RecoHI/HiJetAlgos/python/hiCaloJetsForTrk_cff.py

```diff
@@ -18,4 +18,5 @@
     maxNumber = cms.uint32( 4 )
 )
 
-hiCaloJetsForTrk = cms.Sequence(hiCaloTowerForTrk*akPu4CaloJetsForTrk*akPu4CaloJetsCorrected*akPu4CaloJetsSelected)
+hiCaloJetsForTrkTask = cms.Task(hiCaloTowerForTrk,akPu4CaloJetsForTrk,akPu4CaloJetsCorrected,akPu4CaloJetsSelected)
+hiCaloJetsForTrk = cms.Sequence(hiCaloJetsForTrkTask)
```

Comment:
This sequence is not needed anywhere else, and it can be removed (thus letting around only the corresponding Task)